.github/workflows/validate.yml

```
@@ -75,3 +75,17 @@ jobs:
             pr.txt
             err.txt
             std.txt
+
+      - name: Upload validation errors
+        if: ${{ always() }}
```

```suggestion
        if: failure()
```
It can be only on failures right?

---

now that I combined this to one file this will have to be set to always because warnings will not trigger a failure